src/less/tables.less

```diff
@@ -698,3 +698,59 @@
   }
 }
 
+@media (max-width: @global-screen-small-max) {
+
+  .ReactTable {
+    .rt-table {
+      overflow-x: scroll;
+      margin-left: 102px;
+      width: 80%;
+
+      .rt-tbody {
+        overflow: hidden !important;
+        width: 330px !important;
+      }
+
+      .rt-head {
+        width: 330px !important;
+      }
+
+      .rt-td:nth-child(1),
+      .rt-th:nth-child(1) {
+        z-index: 1;
+        position: absolute !important;
+        width: 100px !important;
+        left: 4px;
+        top: auto;
+        background-color: #fff;
+      }
+
+      .rt-tr {
+        &.-odd {
+          .rt-td:nth-child(1) {
+            background-color: #00000008 !important;
```

Comment:
Too many digits on this hex value. Also can we use a variable? Any of our variables match the color we need here?

Reply:
@juanfabrega I found out the solution for minimizing the use of !important in the code. Will be pushing the updated code. Also, I will replace this with the existing variable we have in variables list.

Reply:
Awesome @rosh-ttnm, thanks! Let me know when it is ready to review. 

As I said earlier, if this is blocking progress, we are ok to move ahead and I have opened a card for future refactoring in JIRA. I will leave the decision up to you.